Tests/selenium/CheckCreateUser.php

```diff
@@ -0,0 +1,147 @@
+<?php
+  /**
+   * Pre-requisites: phpunit, phpunit-selenium, selenium-standalone-server, chrome driver, php-curl extension
+   * 
+   * 
+   * @Matrix Israel Ltd.
+   */
+require_once __DIR__.'/../../vendor/autoload.php';
```

Comment:
I am guessing this is brought in when use composer to bring in phpunit and selenium. is this correct?